ui/src/components/mobileservices/bindingPanelUtils.js

```diff
@@ -75,6 +75,39 @@ function getOpenShiftField(field, properties, idSchema, schema, formData, uiSche
   return getFieldSet(field, properties, idSchema, schema, formData, uiSchema);
 }
 
+function getPasswordField(property, item, idSchema, schema, confirmation) {
+  const id = idSchema[item.key].$id;
+
+  const passwordField = (
+    <div key={`${property.content.key}`}>
+      <label className="control-label" htmlFor={id}>
+        {schema.properties[item.key].title}
+      </label>
+      <input
+        className="form-control"
+        type="password"
+        id={id}
+        onBlur={event => property.content.props.onChange(event.target.value)}
+      />
+    </div>
+  );
+
+  if (confirmation) {
```

Comment:
This might not be needed anymore.

Reply:
I left it since the Template object can be used for other pages as well. In other pages the password confirmation field could make sense.